### DISCUSSION TITLE
Typo on github page

### DISCUSSION THREAD

**Discussion by pahujadeepanshu** [vitmalina/w2ui#2387] [Q&A] [answered]
Hi,

  I am reviewing w2ui for some UI and I just wanted to cross check how stable w2ui 2.0 is. On the landing page I can see a mention about w2ui 2.0.
Can you please confirm if this is a typo?

![image](https://user-images.githubusercontent.com/37369861/220260720-eee0a62a-026a-4ee9-ac64-d429b9fa020a.png)

Thanks,
Deepanshu

**Comment by vitmalina** [comment, marked as answer]
Not a typo, version 2.0 does not need jQuery anymore. However, for backward compatibility, you can still use it with jQuery. For projects w/o jQuery use `w2u.es6.js` file, for projects with jQuery use `w2ui.js` file
